packages/api/__tests__/specs/article.ts

```diff
@@ -55,6 +93,127 @@ describe('Articles', () => {
         }
       })
     })
+
+    test('can be read in list', async () => {
+      const {query} = testClientPrivate
+      const articles = await query({
+        query: ArticleList,
+        variables: {
+          first: 10
+        }
+      })
+      expect(articles).toMatchSnapshot({
+        data: {
+          articles: {
+            nodes: expect.any(Array),
```

Comment:
can you try the following.
`nodes: Array.from({length: articleIDs.length}, () => ({id: expect.any(String)}))`